editor.planx.uk/src/ui/FeaturePlaceholder.tsx

```diff
@@ -0,0 +1,34 @@
+import BuildIcon from "@mui/icons-material/Build";
+import Box from "@mui/material/Box";
+import { styled } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import React from "react";
+
+interface Props {
+  title: string;
+}
+
+const Root = styled(Box)(({ theme }) => ({
+  display: "flex",
+  height: 50,
+  alignItems: "center",
+  background: "white",
+  padding: theme.spacing(4, 3),
+  border: "3px dashed black",
+  borderRadius: "200px",
+  justifyContent: "center",
+}));
+
+const TitleWrap = styled(Box)(() => ({
+  display: "flex",
+  alignItems: "center",
+}));
+
+export const FeaturePlaceholder: React.FC<Props> = ({ title }) => (
+  <Root>
+    <TitleWrap>
+      <BuildIcon sx={{ marginRight: "0.25em" }} />
+      <Typography variant="h4">{title}</Typography>
+    </TitleWrap>
+  </Root>
+);
```

Comment:
Super simple & nice solution :wrench:

Reply:
Thank you 😀

I've just noticed that I'd left some placeholder styling in the component including a hardcoded height that could potentially break with a longer title. Updated to correctly styling.

<img width="531" alt="image" src="https://github.com/theopensystemslab/planx-new/assets/51156018/33a02979-8ae1-445f-986f-8b52c45a6746">